### PR TITLE
upgrade substrate to  558a68f83

### PR DESCRIPTION
**Pull request by brenzi**
* and update to xcm::v1

### CODE REVIEW

**Comment by brenzi** [comment]
this one sucks big time:
https://github.com/paritytech/substrate/commit/6be513d663836c5c5b8a436f5712402a1c5365a3

File: primitives/src/sybil.rs [hunk 1]
```diff
@@ -164,10 +164,10 @@ impl PersonhoodUniquenessRating {
 }
 
 pub fn sibling_junction(id: u32) -> (Junction, Junction) {
-    (Junction::Parent, Junction::Parachain(id))
+    (Junction::OnlyChild, Junction::Parachain(id))
```
Comment (brenzi) [inline primitives/src/sybil.rs:167]: @clangenb can you please check if this is legit? it makes the build work and the test pass, but not sure about the meaning TBH

Comment (brenzi) [reply primitives/src/sybil.rs:167]: hmm, actually, test fails

Comment (clangenb) [reply primitives/src/sybil.rs:167]: hmm, I see that they have changed quite some things, but this seems wrong. I have only checked briefly. I think we should probably use this:

```
MultiLocation {
   parents: 1,
   interior: X1(Parachain(id),
}
```

Comment (brenzi) [reply primitives/src/sybil.rs:167]: I'm actually not sure we need to switch to v1 at all, but I just couldn't make v0 build anymore

Comment (brenzi) [reply primitives/src/sybil.rs]: works with v1 now. you were spot on

**Pull request merged**